### PR TITLE
Resolve some ESLint console warnings

### PR DESCRIPTION
**Pull request by pdehaan**

### CODE REVIEW

**Comment by groovecoder** [comment]
Note: for logging we will eventually adopt [the `mozlog` module](https://www.npmjs.com/package/mozlog) per @jvehent's [logging requirements](https://github.com/mozilla/blurts-server/issues/55).

File: .eslintrc.js [hunk 1]
```diff
@@ -8,14 +8,26 @@ module.exports = {
     "eslint:recommended",
     "plugin:node/recommended",
   ],
+  overrides: [
+    {
+      files: [
+        'scripts/*.js',
+        'tests/**/*.js',
```
[the remark this answers is not in the document]
Comment (nhnt11) [reply .eslintrc.js]: Sure, let's do it.

[the remark this answers is not in the document]
Comment (pdehaan) [reply .eslintrc.js]: What do you all need from me? Did you want me to move the tests/fixtures/ files over, or were you handling that in #108?

Or did you just want me to remove the `'tests/**/*.js'` line above?

**Comment by groovecoder** [comment]
Superseded by https://github.com/mozilla/blurts-server/pull/110

**Pull request closed without merging**